deploy/terraform-aws/main.tf

```diff
@@ -52,33 +54,50 @@ resource "aws_vpc" "main" {
   cidr_block           = var.vpc_cidr
   enable_dns_hostnames = true
 
+  tags = local.k8s_cluster_tags
+}
+
+resource "null_resource" "main" {
+  triggers = {
+    cluster_name = var.cluster_name
+    region = var.region
+  }
+
   provisioner "local-exec" {
     when        = destroy
-    command     = "${path.module}/cleanup-vpc.sh ${self.id} ${var.cluster_name}"
+    command     = "./cleanup-vpc.sh ${self.id} ${self.triggers.cluster_name}"
```

Comment:
The first parameter should be the VPC ID, here the ID of the `null_resource` is passed in.